pkg/kubelet/kubelet_test.go

```diff
@@ -304,7 +306,7 @@ func newTestKubeletWithImageList(
 		HighThresholdPercent: 90,
 		LowThresholdPercent:  80,
 	}
-	imageGCManager, err := images.NewImageGCManager(fakeRuntime, kubelet.StatsProvider, fakeRecorder, fakeNodeRef, fakeImageGCPolicy, "")
+	imageGCManager, err := images.NewImageGCManager(fakeRuntime, kubelet.StatsProvider, fakeRecorder, fakeNodeRef, fakeImageGCPolicy, "", tracing.NewNoopTracerProvider())
```

Comment:
Please use `go.opentelemetry.io/otel/trace.NewNoopTracerProvider()` here and in pkg/kubelet/kuberuntime/kuberuntime_manager_test.go.  The component-base tracing NoOp is only needed if you are handling kubelet-wide shutdown within the package, which shouldn't happen within the image manager or kuberuntime manager.

Reply:
Good idea, done